src/Compilers/Core/Portable/SourceGeneration/GeneratorDriverState.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+
+#nullable enable
+namespace Microsoft.CodeAnalysis
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("ApiDesign", "RS0016:Add public types and members to the declared API", Justification = "<Pending>")]
+    public readonly struct GeneratorDriverState
```

Comment:
>public [](start = 4, length = 6)

Why is this a public type, it looks like it is not mentioned in any public API and it doesn't have any public members?